app/assets/javascripts/modules/ResultsFilter.js

```diff
@@ -3,34 +3,55 @@ define(['jquery'], function($) {
   'use strict';
 
   var $resultsFilter = $('[data-results-filter]'),
+
       $resultsFilterTrigger = $('[data-results-filter-trigger]'),
-      $resultsFilterTriggerOne = $('[data-results-filter-trigger="1"]'),
-      $resultsFilterTriggerTwo = $('[data-results-filter-trigger="2"]'),
+      $resultsFilterTriggerOne   = $('[data-results-filter-trigger="1"]'),
+      $resultsFilterTriggerTwo   = $('[data-results-filter-trigger="2"]'),
+      $resultsFilterTriggerThree = $('[data-results-filter-trigger="3"]'),
+
       $resultsFilterTarget = $('[data-results-filter-target]'),
-      $resultsFilterTargetOne = $('[data-results-filter-target="1"]'),
-      $resultsFilterTargetTwo = $('[data-results-filter-target="2"]'),
+      $resultsFilterTargetOne   = $('[data-results-filter-target="1"]'),
+      $resultsFilterTargetTwo   = $('[data-results-filter-target="2"]'),
+      $resultsFilterTargetThree = $('[data-results-filter-target="3"]'),
+
+      $resultsFilterTargetOneInput   = $('[data-results-input-target="1"]'),
+      $resultsFilterTargetThreeInput = $('[data-results-input-target="3"]'),
```

Comment:
This is getting a bit repetitive and messy - I think we should refactor it slightly. Items 1 and 3 seem to do the same thing (reveal their sibling input field) so they can have a shared action.